src/components/DraftSection.vue

```diff
@@ -128,11 +134,95 @@ const heroChosen = (hero: IHero | null) => {
   state.selectedHero = null;
 };
 
+const timerStart = () => {
+  window.clearInterval(timerStore.radiantTimerInterval);
+  window.clearInterval(timerStore.direTimerInterval);
+  timerStore.setTimeByDefault(TimerMapper.RADIANT_MAIN_TIME);
+  timerStore.setTimeByDefault(TimerMapper.DIRE_MAIN_TIME);
+
+  if (state.roundCounter > DEFAULT_NUMBER_OF_ROUNDS) return;
+
+  if (
+    FIRST_PICK_PICK_ROUNDS.includes(state.roundCounter) ||
+    FIRST_PICK_BAN_ROUNDS.includes(state.roundCounter)
+  ) {
+    if (configStore.isRadiantFirstPick) {
+      timerStore.radiantTimerInterval = window.setInterval(() => {
```

Comment:
`setInterval` instead of `window.setInterval`

Reply:
![image](https://user-images.githubusercontent.com/73604138/226166204-e4025fb4-548b-41ec-b427-06a8d21e6d25.png)